src/main/cliapp/Makefile

```diff
@@ -4,6 +4,12 @@ all:
 run:
 	npm start
 
+run-alpha:
+	npm run start-alpha
+
+run-local:
+	npm run start-local
+
```

Comment:
@adamgibs probably want to remove both these lines also as the scripts are now removed.